src/App.tsx

```diff
@@ -1,12 +1,23 @@
+import { GoogleOAuthProvider } from '@react-oauth/google';
+import { QueryClient, QueryClientProvider } from 'react-query';
+import { createBrowserRouter, RouterProvider } from 'react-router-dom';
```

Comment:
Can we fix the imports in specific order
1) Library imports
2) Component imports
3) helpers/css